content/en/blog/2019/dns-cert/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Istio DNS certificate management
+description: Provision and manage DNS certificates in Istio.
+publishdate: 2019-11-08
+attribution: Lei Tang (Google)
+keywords: [security, kubernetes, certificates, DNS]
+target_release: 1.4
+---
+
+By default, Citadel manages the DNS certificates of the Istio control plane.
+Citadel is a large component that maintains its own private signing key, and acts as a Certificate Authority (CA).
+
+New in Istio 1.4, we introduce a feature to securely provision and manage DNS certificates
+signed by the Kubernetes CA, which has the following advantages.
+
+* Lighter weight DNS certificate management with no dependency on Citadel.
+
+* Unlike Citadel, this feature doesn't maintain a private signing key, which enhances security.
+
+* Simplified root certificate distribution to TLS clients.
+Clients no longer need to wait for Citadel to generate and distribute its CA certificate.
```

Comment:
what is the clients here?   sidecar proxy or istio control plane components?

Reply:
or is this citadel agent?

Reply:
Another thing that would be useful is to mention if I use chiron, do i still need citadel?  if so, why?

Reply:
The clients here are istio control plane components. If you use Chiron to provision control plane DNS certificates, you do not need Citadel to provision control plane certificates.